src/components/Markdown/Markdown.tsx

```diff
@@ -207,7 +207,7 @@ export class Markdown extends React.PureComponent<MarkdownProps, { html }> {
         return source;
     }
 
-    UNSAFE_componentWillReceiveProps(next_props) {
+    componentDidUpate(next_props) {
         if (next_props.source !== this.props.source) {
```

Comment:
Hmm, this is a weird one then. If we update `next_props` to become `this.props` as above then we end up with `this.props.source !== this.props.source`, which seems unlikely to be correct :) Maybe I need to better understand how this is working.

Reply:
The comparison is still valid I think, it's just comparing with `this.props` as being the current value instead of the old value, and the parameter being passed in is now the old value instead of the value to be.